test/e2e/common_suite.go

```diff
@@ -16,34 +16,31 @@ import (
 	log "github.com/sirupsen/logrus"
 	v1 "k8s.io/api/core/v1"
 	"sigs.k8s.io/e2e-framework/pkg/env"
-	envconf "sigs.k8s.io/e2e-framework/pkg/envconf"
 )
 
+const E2eNamespace = "e2e-test"
```

Comment:
Can this be project specific i.e. `coco-e2e-test` or `coco-pp-e2e-test` ?

Reply:
Good idea. I'll update that now. Thanks!